src/fprime/fbuild/settings.py

```diff
@@ -148,6 +148,9 @@ def load(settings_file: Path, cwd: Path):
             "install_dest": install_dest,
             "environment_file": env_file,
             "environment": environment,
+            "cookiecutter": confparse.get(
+                "fprime", "cookiecutter", fallback="native"
```

Comment:
What about using `default` or `builtin` instead of native for the default value?

Reply:
Ok sounds good, but could give a quick explanation to the benefits of this change because I'm not sure I understand the difference?

Reply:
We use native to mean a native build, as in not-cross compiled and you're building for your local computer. Native just doesn't seem to make sense in the context of selecting which cookiecutter template to use

Reply:
Ok makes sense thanks!